src/riak_kv_pb_timeseries.erl

```diff
@@ -64,6 +64,7 @@
 -define(E_MISSING_TS_MODULE, 8).
 -define(E_DELETE,   9).
 -define(E_GET,     10).
+-define(E_BAD_KEY_LENGTH, 11).
```

Comment:
I suggest that we define TS error codes to be in a certain range that is outside of 1-1000. There will be work done soon to improve error reporting to Riak clients and each application within Riak will use its own set of codes. See RIAK-1437 in JIRA.